.eslintrc.json

```
@@ -31,6 +31,7 @@
     }
   },
   "rules": {
+    "eslint-comments/require-description": "error",
```

This rule will make mandatory to comment on any ESLint directive. However, I do not think that we should comment when re-enabling a disabled rule. Since I do not think anyone would enable an extra rule, it should be safe.

```suggestion
    "eslint-comments/require-description": ["error", {"ignore": ["eslint-enable"]}],
```